apps/ledger-live-mobile/src/families/ethereum/EthereumCustomFees/utils.tsx

```diff
@@ -31,7 +31,9 @@ export const inferMaxFeeRange = (
     return defaultMaxFeePerGasRange;
 
   const { maxPriorityFeePerGas, nextBaseFeePerGas } = networkInfo;
-  const amplifiedBaseFee = nextBaseFeePerGas.times(2); // ensuring 6 blocks of base fee
+  const amplifiedBaseFee = nextBaseFeePerGas.times(
+    getEnv("EIP1559_BASE_FEE_MULTIPLIER"),
+  ); // ensuring 6 blocks of base fee
```

Comment:
Comment could be updated I guess, nbd (others as well)

Reply:
Yup, missed this one, updating it 👍

Reply:
Done